src/MongoDB.Driver.Core/Core/Bindings/CoreServerSession.cs

```diff
@@ -47,6 +48,9 @@ public CoreServerSession()
         }
 
         // public properties
+        /// <inheritdoc />
+        public bool Dirty => _dirty;
```

Comment:
Minor question: should this property be named `IsDirty` to conform to our naming standards?

Reply:
Done